### PR TITLE
Update xray docs

### PR DESCRIPTION
**Pull request by vnandwana**
Added steps to install xray daemon on standalone EC2.

### CODE REVIEW

File: userguide/aws/how-to-set-up-a-cloud-formation-system.adoc [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
 This document describes the procedures for setting up Kill Bill under AWS using the CloudFormation option. This is one of two recommended alternatives for production use. The multi-tier option requires more setup than CloudFormation, but provides more control over the deployment. The procedures in this document are based on the options recommended by Kill Bill.
 
-Deployment via CloudFormation leverages the capabilites of the AWS infrastructure to provide a robust *production ready* deployment with a single click. The entire CloudFormation configuration, or stack, is defined by a *CloudFormation Template*.
+Deployment via CloudFormation leverages the capabilities of the AWS infrastructure to provide a robust *production ready* deployment with a single click. The entire CloudFormation configuration, or stack, is defined by a *CloudFormation Template*.
 
 The features of the CloudFormation system include:
 
-* Both Kill Bill and Kaui instances can be scaled up or down using AWS autoscaling groups
+* Kill Bill and Kaui instance can be scaled up or down using AWS autoscaling groups
```
Comment (reshmabidikar) [inline userguide/aws/how-to-set-up-a-cloud-formation-system.adoc]: Consider rephrasing as 
```suggestion
*The instance having Kill Bill/Kaui installed can be scaled up or down using AWS autoscaling groups
```

File: userguide/aws/how-to-set-up-a-cloud-formation-system.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ image::../assets/aws/cf_stack.png[align=center]
 
 All resources for this system run within a single AWS *Virtual Private Cloud (VPC)*, providing a dedicated block of IP addresses which must be located in a single *region*. The cloud is partitioned into *availability zones*, which are accessed by *subnets*. The resources must be distributed over at least two availability zones.
 
-Multiple instances of Kaui and the Kill Bill server are deployed in the VPC, each running on its own Ubuntu Linux server. AWS autoscaling is used to increase and decrease the number of instances for each package as needed.
+A single instance of Kill Bill and Kaui servers is deployed in the VPC, running on an Ubuntu Linux server. AWS autoscaling is utilized to dynamically adjust the number of instances for each package based on demand, allowing for efficient scaling.
```
Comment (reshmabidikar) [inline userguide/aws/how-to-set-up-a-cloud-formation-system.adoc]: Consider rephrasing as 
```suggestion
A single instance having the Kill Bill/Kaui server is deployed in the VPC, running on an Ubuntu Linux server. AWS autoscaling is utilized to dynamically adjust the number of instances for each package based on demand, allowing for efficient scaling.
```

**Review by reshmabidikar — APPROVED** [review]
Looks good, left some minor comments.

**Comment by vnandwana** [comment]
@reshmabidikar I have rebased this branch to v3 but those changes are not reflected in this PR. I'm closing this PR and have opened a new one -- [https://github.com/killbill/killbill-docs/pull/540](https://github.com/killbill/killbill-docs/pull/540). Kindly take a look at it when you get a chance. Thank you.

**Pull request closed without merging**